packages/ui/components/top-banner/topBanner.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { vi } from "vitest";
+
+import { ArrowDown } from "@calcom/ui/components/icon";
+
+import { TopBanner } from "./TopBanner";
+
+describe("Tests for TopBanner component", () => {
+  test("Should render the component properly", () => {
+    render(<TopBanner text="the banner test" />);
+
+    const bannerElt = screen.getByTestId("banner");
+    expect(bannerElt).toBeInTheDocument();
+
+    const btnElt = screen.queryByRole("button");
+    expect(btnElt).toBeNull();
+  });
+
+  test("Should render a button", () => {
+    const mockOnClose = vi.fn();
+    render(<TopBanner text="the banner test" onClose={mockOnClose} />);
+
+    const btnElt = screen.getByRole("button");
+    expect(btnElt).toBeInTheDocument();
+
+    fireEvent.click(btnElt);
+    expect(mockOnClose).toHaveBeenCalled();
```

Comment:
Hello! As you can see there is a test failing and this is caused by the `onClose` prop not being called in the component, what can we do about this prop?

Reply:
Let's remove the functionality. I don't think anywhere we are passing onClose prop to it. Things are working through `actions` prop